src/main/java/com/azure/cosmos/sample/sync/SyncMain.java

```diff
@@ -0,0 +1,164 @@
+        familiesToCreate.add(Families.getJohnsonFamilyDocument());
+        familiesToCreate.add(Families.getSmithFamilyDocument());
+
+        createFamilies(familiesToCreate);
+
+        System.out.println("Querying documents.");
+        queryDocuments();
+    }
+
+    private void createDatabaseIfNotExists() throws Exception {
+        System.out.println("Create database " + databaseName + " if not exists.");
+
+        //  Create database if not exists
+        database = client.createDatabaseIfNotExists(databaseName).getDatabase();
+
+        System.out.println("Checking database " + database.getId() + " completed!\n");
+    }
+
+    private void createContainerIfNotExists() throws Exception {
+        System.out.println("Create collection " + containerName + " if not exists.");
+
+        //  Create collection if not exists
+        CosmosContainerProperties containerProperties =
+            new CosmosContainerProperties(containerName, "/lastName");
+
+        //  Create container with 400 RU/s
+        container = database.createContainerIfNotExists(containerProperties, 400).getContainer();
+
+        System.out.println("Checking collection " + container.getId() + " completed!\n");
+    }
+
+    private void createFamilies(List<Family> families) throws Exception {
+        double totalRequestCharge = 0;
+        for (Family family : families) {
+
+            //  Create item using container that we created using sync client
+
+            //  Use lastName as partitionKey for cosmos item
+            //  Using appropriate partition key improves the performance of database operations
+            CosmosItemRequestOptions cosmosItemRequestOptions = new CosmosItemRequestOptions(family.getLastName());
+            CosmosItemResponse item = container.createItem(family, cosmosItemRequestOptions);
+
+            //  Get request charge and other properties like latency, and diagnostics strings, etc.
+            System.out.println(String.format("Created document with request charge of %.2f within" +
+                    " duration %s",
+                item.getRequestCharge(), item.getRequestLatency()));
+            totalRequestCharge += item.getRequestCharge();
+        }
+        System.out.println(String.format("Created %d documents with total request " +
+                "charge of %.2f",
+            families.size(),
+            totalRequestCharge));
+    }
+
+    private void queryDocuments() {
+        // Set some common query options
+        FeedOptions queryOptions = new FeedOptions();
+        queryOptions.maxItemCount(10);
+        queryOptions.setEnableCrossPartitionQuery(true);
+        //  Set populate query metrics to get metrics around query executions
+        queryOptions.populateQueryMetrics(true);
+
+        Iterator<FeedResponse<CosmosItemProperties>> feedResponseIterator = container.queryItems(
+            "SELECT * FROM Family WHERE Family.lastName != 'Andersen'", queryOptions);
```

Comment:
Instead of a cross-partition query, can we do a targeted partition SQL query like, SELECT * WHERE c.lastName IN ("Anderson", "Wakefield") for the query sample? Then we can mention that choosing a partition key we often use in the where filter leads to more efficient queries.
Though users do sometimes need blanket cross-partition queries in their workloads, putting one in the quickstart may lead to the impression it's a good thing to do upfront. If we can, we should try to impart the importance of having a good partition key based on the queries early on :)

```suggestion
            "SELECT * FROM Family WHERE Family.lastName IN ('Andersen', 'Wakefield')", queryOptions);
```

I'm not sure - do we still need setEnableCrossPartitionQuery(true) if the query targets select partitions?